src/test/java/net/sf/jabref/remote/DBConnectorTest.java

```diff
@@ -0,0 +1,58 @@
+package net.sf.jabref.remote;
+
+import java.sql.SQLException;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class DBConnectorTest {
+
+    @Test
+    public void testGetNewMySQLConnection() {
+        try {
+            DBConnector.getNewConnection(DBType.MYSQL, "localhost", "jabref", "root", "");
```

Comment:
The problem is that `DriverManager.getConnection(url, user, password);` in `DBConnector` never returns `null`. Unfortunately there is no opposite method to `Assert.fail()`. A test method passes when no fails occur.